prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullname = 'Aldwin Corpus aka Ace Corpus';
+    console.log('Value of fullname:', fullname);
+
+    var numberOfPets = 0;
+    console.log('Value of numberOfPets:', numberOfPets);
+
+    var likesVegetables = true < false;
```

Comment:
You can leave out the `<` and just pick `true` or `false`.

Reply:
Got it! Let me send do it again

Reply:
done